File: DemoApplication.Web/src/app/models/user/user.model.ts
import { SignInModel } from "../auth/signIn.model";

// this class defines a user and it's properties
export class UserModel {
    constructor(
    public id?: number,
    public name?: string,
    public surname?: string,
    public email?: string,
    public active?: boolean,
    public auth?: SignInModel
    )   {
    }
}

File: DemoApplication.Models/User/UserModel.cs
using System;

namespace DemoApplication.Models;

public class UserModel
{
    public Guid Id { get; set; }

    public string Name { get; set; } = string.Empty;

    public string Surname { get; set; } = string.Empty;

    public string Email { get; set; } = string.Empty;

    public bool Active { get; set; }

    public AuthModel Auth { get; set; } = new();
}
